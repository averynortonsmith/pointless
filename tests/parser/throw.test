
x = throw ParserError
x = throw 1 + 2