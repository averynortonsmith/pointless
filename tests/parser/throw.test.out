(Program 
 null
 []
 (Def 
  (Name "x")
  (Throw (Label "ParserError")))
 (Def 
  (Name "x")
  (Throw 
   (BinaryOp 
    Tok.Add
    (Number 1)
    (Number 2)))))
